src/Block/Page/Footer.php

```diff
@@ -99,7 +94,7 @@ public function getScandiPWADirectoryPath() {
         $themeDirectoryPath = null;
 
         foreach ($this->themeList as $theme) {
-            if ($theme->getFullPath() === self::SCANDIPWA_COMPONENT_NAME) {
+            if (str_contains($theme->getFullPath(), "frontend/scandipwa")) {
```

Comment:
Please use single quotes for strings.

Reply:
Please move `frontend/scandipwa` to the class constant.

Reply:
done